src/pages/health-worker-status.page.js

```diff
@@ -70,8 +71,16 @@ class HealthWorkerStatusPage extends Component {
     this.setState({ healthWorkerStatus });
     this.setState({ isSelected: event.isActive });
 
-    // todo: put this in proper app state place
-    sessionStorage.setItem('healthWorkerStatus', JSON.stringify(selectedStatus));
+    setAppState({
+      ...appState,
+      profile:{
+        ...appState.profile,
+        options:{
+          ...appState.profile.options,
+          healthWorkerStatus: selectedStatus
```

Comment:
options is just a list of all the possible keys for that select. i think you will need a new key here